tools/ci_static_check_reports/build_static_checkers_reports.py

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+import os
+import sys
+from pathlib import Path
+from typing import Dict, List
+
+from invoke import Context
+
+sys.path.insert(0, "airbyte-integrations/connectors")
+from tasks import CONFIG_FILE, TOOLS_VERSIONS, _run_task  # noqa
+
+TASK_COMMANDS: Dict[str, List[str]] = {
+    "black": [
+        f"pip install black~={TOOLS_VERSIONS['black']}",
+        f"XDG_CACHE_HOME={os.devnull} black -v {{check_option}} --diff {{source_path}}/. > {{reports_path}}/black.txt",
+    ],
+    "coverage": [
+        "pip install .",
+        f"pip install coverage[toml]~={TOOLS_VERSIONS['coverage']}",
+        "coverage xml --rcfile={toml_config_file} -o {reports_path}/coverage.xml",
+    ],
+    "flake": [
+        f"pip install mccabe~={TOOLS_VERSIONS['mccabe']}",
+        f"pip install pyproject-flake8~={TOOLS_VERSIONS['flake']}",
+        f"pip install flake8-junit-report~={TOOLS_VERSIONS['flake_junit']}",
+        "pflake8 -v {source_path} --output-file={reports_path}/flake.txt --bug-report",
```

Comment:
Can we move all dependencies to the package setup.py file? or do a bit of tools have deps conflicts?

Reply:
these are deps, used by `tasks.py`
these are not `ci_static_check_reports` package requirements.